L1Trigger/L1TTrackMatch/interface/L1TkHTMissEmulatorProducer.h

```diff
@@ -38,10 +39,10 @@ namespace l1tmhtemu {
   typedef ap_int<kInternalPhiWidth> phi_t;
 
   typedef ap_int<kInternalPtWidth + kEtExtra> Et_t;
```

Comment:
According to the interface document the scalar sum should also be an `ap_ufixed<16,11>`. Doesn't that mean `Et_t` should be an `ap_ufixed`?

Reply:
This is an internal integer datatype that is used in the cordic functions for both MET and MHT and allows us to increase the precision of this internal datatype for studying the affect of that. This is then shifted at the end of the calculation for the correct interface specification

Reply:
But the HT also uses the `Et_t` type and it is not converted or its precision reduced before being added to the output. So if you don't want to change the `Et_t` type, then the HT value needs to be modified before it reaches the output. Right now it's using 26 bits in the output word.

Reply:
Fixed, switched the HT to use the ap_ufixed<16,11> datatype